src/views/LogoutLedger.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+    <div class="container">
+        <SmallPage>
+            <PageHeader>Mind the loss of settings</PageHeader>
```

Comment:
Is setting a good word? Please check with Julian, because for me "addresses and labels" are not _settings_ really.

Reply:
All the wording is coming from Julian / Tammo.